mandatory/3-function-output.js

```diff
@@ -1,16 +1,20 @@
-// Add comments to explain what this function does. You're meant to use Google!
+
+//This function returns a random number between 0 and 10
 function getRandomNumber() {
   return Math.random() * 10;
 }
```

Comment:
"The Math.random() function returns a floating-point, pseudo-random number in the range 0 to less than 1 (inclusive of 0, but not 1) with approximately uniform distribution over that range — which you can then scale to your desired range."

The above description is taken from MDN Docs. According to this, the `Math.random()` function randomly generates a number between 0 and 1, then multiplies it with a number of your choice.